utility_network/configure-subnetwork-trace/src/main/java/module-info.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Esri.
+ * Copyright 2022 Esri.
```

Comment:
You should really create the module-info.java as a new file and delete the Launcher as it's a cleaner story.

Reply:
Hi @jenmerritt, I created this as a new file and deleted launcher however, I am not sure why the code shows mixed with the launcher file in Git when reviewing.

Reply:
Ok, if it's Github rather than anything you did we'll just leave it!